CRM/Case/BAO/Case.php

```diff
@@ -529,6 +529,9 @@ public static function retrieveCaseIdsByContactId($contactID, $includeDeleted =
     if (!$includeDeleted) {
       $query .= " AND ca.is_deleted = 0";
     }
+    if (isset($statusId)) {
+      $query .= " AND ca.status_id = $statusId";
+    }
```

Comment:
I believe status id is stored as an integer in the database. I'm not entirely sure but is it necessary to escape int variables?